src/mpid/ch4/netmod/ofi/ofi_am_impl.c

```diff
@@ -39,6 +40,10 @@ int MPIDI_OFI_deferred_am_isend_issue(MPIDI_OFI_deferred_am_isend_req_t * dreq)
     need_packing = dt_contig ? false : true;
     send_buf = (char *) dreq->buf + dt_true_lb;
 
+    data_lt_eager_threshold =
```

Comment:
what is "lt"?

Reply:
Oh, "less than". Can we change the name of this to `bool do_eager`?

Reply:
Another nit, this is not a really a fix and could be split out.

Reply:
True, this is not a real fix. It is more of avoiding long calculation lines.

Reply:
I have renamed it.